htdocs/compta/paiement.php

```diff
@@ -449,19 +450,22 @@ function callForResult(imgId)
 		}
 
 		print '<form id="payment_form" name="add_paiement" action="'.$_SERVER["PHP_SELF"].'" method="POST">';
-		print '<input type="hidden" name="token" value="'.newToken().'">';
+		print '<input type="idden" name="token" value="'.newToken().'">';
 		print '<input type="hidden" name="action" value="add_paiement">';
 		print '<input type="hidden" name="facid" value="'.$facture->id.'">';
 		print '<input type="hidden" name="socid" value="'.$facture->socid.'">';
 		print '<input type="hidden" name="type" id="invoice_type" value="'.$facture->type.'">';
 		print '<input type="hidden" name="thirdpartylabel" id="thirdpartylabel" value="'.dol_escape_htmltag($facture->thirdparty->name).'">';
 
-		print dol_get_fiche_head();
 
-		print '<table class="border centpercent">';
+		print '<div class="fichecenter">';
+		print '<div class="fichehalfleft">';
+		print '<div class="underbanner clearboth"></div>';
+	
+		print '<table class="noborder tableforfield" centpercent>';
 
 		// Third party
-		print '<tr><td class="titlefieldcreate"><span class="fieldrequired">'.$langs->trans('Company').'</span></td><td>'.$facture->thirdparty->getNomUrl(4)."</td></tr>\n";
+		print '<tr><td width=30% class="fieldrequired">'.$langs->trans('Company').'</td><td>'.$facture->thirdparty->getNomUrl(4)."</td></tr>\n";
```

Comment:
Use of percent is deprecated. You must use a css style line "titlefieldcreate" for example;